src/docs/stan-reference/examples.tex

```diff
@@ -7000,12 +7000,10 @@ \section{Simulating from a Gaussian Process}
   real x[N];
 }
 transformed data {
-  vector[N] mu;
-  cov_matrix[N] K;
-  mu = rep_vector(0, N);
-  K = cov_exp_quad(x, 1, 1);
-  for (i in 1:N)
-    K[i, i] = K[i, i] + 0.1;
+  matrix[N, N] K = cov_exp_quad(x, 1.0, 1.0);
```

Comment:
I'm guessing it's to avoid having an issue with the `cov_matrix` constraint check being too strict.  I think it's better  behaved now.  I'd rather see this declared as `cov_matrix` if that's computationally stable and documented if it's not (yet).